cli/tests/test_workspaces.rs

```diff
@@ -1055,21 +1097,39 @@ fn test_colocated_workspace_update_stale() {
     let output = main_dir.run_jj(["workspace", "update-stale"]);
     insta::assert_snapshot!(output, @r"
     ------- stderr -------
-    Working copy  (@) now at: rlvkpnrz f56876af (empty) (no description set)
-    Parent commit (@-)      : qpvuntsm 5ed82d60 (no description set)
+    Abandoned 1 commits that are no longer reachable.
+    Done importing changes from the underlying Git repo.
+    Concurrent modification detected, resolving automatically.
+    Working copy  (@) now at: rlvkpnrz f562bf82 (empty) (no description set)
+    Parent commit (@-)      : qpvuntsm 30ed2f28 (no description set)
     Added 0 files, modified 1 files, removed 0 files
-    Updated working copy to fresh commit f56876af92ff
+    Updated working copy to fresh commit f562bf82f2da
     [EOF]
     ");
 
-    // Verify the workspace is now up-to-date
+    // Verify the workspace is now up-to-date. New bookmark "book2" should have
+    // been imported by the previous command.
     let output = main_dir.run_jj(["st"]);
     insta::assert_snapshot!(output, @r"
     The working copy has no changes.
-    Working copy  (@) : rlvkpnrz f56876af (empty) (no description set)
-    Parent commit (@-): qpvuntsm 5ed82d60 (no description set)
+    Working copy  (@) : rlvkpnrz f562bf82 (empty) (no description set)
+    Parent commit (@-): qpvuntsm 30ed2f28 (no description set)
     [EOF]
     ");
+
+    // FIXME: The updated bookmark "book1" shouldn't be re-imported.
```

Comment:
nit: I found the version of this comment on the first commit quite confusing. Something like 'The "old book1" should commit should still be visible' would be clearer.